components/profile_popover/index.js

```diff
@@ -15,6 +18,8 @@ function mapStateToProps(state) {
     return {
         enableWebrtc,
         enableTimezone: areTimezonesEnabledAndSupported(state),
+        currentUserId: getCurrentUserId(state),
+        teamUrl: '/' + getCurrentTeam(state).name,
```

Comment:
Does this handle subpaths properly?

Reply:
It should, the subpath is handled by webpack and is transparent to the JS code. This is also effectively doing what was occuring before, there's no functional change